data/models/articles/A_Friendly_Tool_For_React_Native_Projects/info.json

```
@@ -0,0 +1,5 @@
+{
+  "title": "A Friendly Tool For React Native Projects",
+  "description": "How this tool was created and what it can be helpful for?",
+  "image": "https://miro.medium.com/max/1400/1*nmzHh215x4FgBqm8RDPjug.png"
+}
```

don't forget to credit yourself ;)

```suggestion
  "image": "https://miro.medium.com/max/1400/1*nmzHh215x4FgBqm8RDPjug.png"
  "authors": ["Omar-Belghaouti"]
}
```